web/src/main/webapp/WEB-INF/jsp/submit/addNewSubject.jsp

```diff
@@ -266,10 +260,6 @@
            <span class="formlabel">&nbsp;*</span>
         </c:when>
         </c:choose>
-        <c:if test="${study.studyParameterConfig.discrepancyManagement=='true'}">
-	        <a href="#" onClick="openDSNoteWindow('CreateDiscrepancyNote?name=subject&field=gender&column=gender','spanAlert-gender'); return false;">
-	        <img name="flag_gender" src="images/icon_noNote.gif" border="0" alt="<fmt:message key="discrepancy_note" bundle="${resword}"/>" title="<fmt:message key="discrepancy_note" bundle="${resword}"/>"></a>
-	    </c:if>
 	</td>
```

Comment:
It is unclear whether the ability to add Queries during subject creation should be removed.  It works in 3.x if you want to bypass a required field but still create the subject.

Product Owner to follow up and determine if Queries should be removed from this page.